### ISSUE TITLE
Running synthesis in arctic example rms outdir

### ISSUE THREAD

**Issue by matthewaylett** [Idlak/idlak#107]
If you run the example synthesis command pointing at . it deleted everything without prompting. I accidentally scrubbed the whole s2 directory.

**Comment by ninaburns** [comment]
Yep.  This happened to me when running the example synthesis command for tts-tangle-lj.
